src/exampleTypes/index.js

```diff
@@ -26,7 +26,7 @@ export default (
   let Components = {
     Facet: partial({ TextInput, Checkbox, RadioList }, Facet),
     Number: partial({ NumberInput }, Number),
-    Date: partial({ DateInput }, Date),
+    Date: partial({ DateInput }, DateComponent),
```

Comment:
The key here needs to change too because there is no `Components.DateComponent,` on line 43.